gen/definitions/dynamic_object.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: DynamicObject
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/object/dynamicobjects
+data_source_name_query: true
+doc_category: Objects
+attributes:
+  - model_name: name
+    type: String
+    mandatory: true
+    description: User chosen resource name.
+    example: DynamicObject1
+  - model_name: type
+    type: String
+    description: Type of the Dynamic objects; this value is always DynamicObject.
+    value: DynamicObject
+  - model_name: description
+    type: String
+    description: The details about the Dynamic Object.
+    example: My dynamic object
+  - model_name: objectType
+    mandatory: true
+    type: String
+    description: Type of dynamic object mappings. Currently we support only 'IP'.
+    example: IP
+  - model_name: agentId
+    type: String
+    description: The agent ID Dynamic Object.
+    exclude_test: true
+    example: agent_007
+  - model_name: topicName
+    type: String
+    description: Topic of the Dynamic Object.
+    exclude_test: true
+    example: aws365
```

Comment:
Adding two attributes agentId and topicName but without tests. Hmm. Let's try to minimize dead code. I looked closer and these attributes do not show up in webUI 7.2.5 and they are not needed for feature parity. I think either:

1. (my suggestion) remove them for now
2. (not my suggestion) alternatively make them complete by figuring out:
  a. do these attribs bring user-facing benefits? They might be weird artifacts of the api spec.
  b. and also: how to test them?